tests/test_forecast.py

```diff
@@ -12,10 +12,26 @@ def test_run_forecast():
     predications_df_gfs = run_forecast(site=site, ts=ts, nwp_source="gfs")
     predications_df_icon = run_forecast(site=site, ts=ts, nwp_source="icon")
 
-    print("\nPrediction based on GFS NWP\n")
+    print("\n Prediction based on GFS NWP\n")
     print(predications_df_gfs)
-    print(f"Max: {predications_df_gfs['power_wh'].max()}")
+    print(f" Max: {predications_df_gfs['power_wh'].max()}")
 
-    print("\nPrediction based on ICON NWP\n")
+    print("\n Prediction based on ICON NWP\n")
     print(predications_df_icon)
-    print(f"Max: {predications_df_icon['power_wh'].max()}")
+    print(f" Max: {predications_df_icon['power_wh'].max()}")
+
+    ts = datetime.today() - timedelta(days=200)
+
+    # run model with icon and gfs nwp
```

Comment:
do you recond this could be split into a different test? 
Then we know if `now` and `historic` tests are working or now

Reply:
What do you think about this splitting? Or should we add this also to another test file?
Did i need to add the test function somewhere additionally ?

Reply:
I was thinking keeping it in the same file, but just have two "test" functions. Then pytest runs them seperately

Reply:
I see you have done that - thanks